src/components/PopupModal/PopupModal.tsx

```diff
@@ -0,0 +1,133 @@
+import React, {
+  ReactNode,
+  ComponentPropsWithoutRef,
+  useState,
+  useEffect,
+  cloneElement,
+  Children,
+} from "react";
+import { FiX } from "react-icons/fi";
+
+interface IPopupModalProps extends ComponentPropsWithoutRef<"div"> {
+  status?: "hidden" | "displayed" | undefined;
+  fullMode?: boolean;
+  children?: ReactNode;
+
+  // both exposed function should be declared as:
+  // onOpen?(): any;
+  // however by some reason jest does not recognize
+  onOpen?: () => void;
+  onClose?: () => void;
+}
+
+interface IPopupModalNodeProps extends ComponentPropsWithoutRef<"div"> {
+  children?: ReactNode;
+  _onClose?: () => void;
+}
+
+export function PopupModalHeader(props: IPopupModalNodeProps) {
+  const { children, _onClose } = props;
+
+  function handleClose() {
+    _onClose?.();
+  }
+
+  if (children) {
+    return (
+      <div
+        data-testid="popup-modal-header"
+        className="flex items-start justify-between p-4 rounded-t"
+      >
+        <h3 className="text-xl font-semibold text-f-primary">{children}</h3>
```

Comment:
` font-semibold ` do we not have a corresponding font in the presets ?

Reply:
Good catch! 

In fact we don't need that since is a children and we can customized that!

Reply:
```typescript
".mas-h3": {
          fontSize: "14px",
          fontWeight: "500",
          fontFamily: theme("fontFamily.Urbane"),
          lineHeight: "16px",
        },
``` 

Sure! We have a className for `h3` tag.